script/install/script.rb

```diff
@@ -6,6 +6,8 @@
 
 TARGET_VIEWS_PATH = Pathname.new('app/views/start_bootstrap_admin_theme/pages/')
 START_BOOTSTRAP_PATH = Pathname.new(ARGV[0])
+FONTAWESOME_RELATIVE_PATH = 'vendor/fontawesome-free/css'
```

Comment:
@bigxiang Hmmm.. It's a bit more convenient to keep `fontawesome` for the path, since that's the naming used in the theme package download. For convenience, the script retains the directory structure of the theme package when it imports its files to the vendor directory. 

However, I don't mind changing the constant to `FONT_AWESOME_RELATIVE_PATH`. What do you think?

Reply:
It's not a big deal, just choose the name you feel convenient. :)